modules/storage-persistent-storage-efs-provisioner.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-efs.adoc
+
+[id="efs-provisioner_{context}"]
+= Create the EFS Provisioner
+
+The EFS provisioner is an {product-title} Pod that mounts the EFS volume
+as an NFS share.
+
+.Prerequisites
+
+* A ConfigMap has been created that defines the EFS environment variables.
+* A service account has been defined that contains the necessary cluster
+and role permissions.
+* A StorageClass has been created for provisioning volumes.
+* The Amazon security groups have been configured to allow incoming NFS
+traffic on all {product-title} nodes.
+* The Amazon EFS volume security group has been configured to allow incoming
+SSH traffic from all sources.
+
+.Procedure
+
+. Define the EFS provisoiner by creating a `provisioner.yaml` with the
+following contents:
++
+[source,yaml]
+----
+kind: Pod
+apiVersion: v1
+metadata:
+  name: efs-provisioner
+spec:
+  serviceAccount: efs-provisioner
+  containers:
+    - name: efs-provisioner
+      image: quay.io/external_storage/efs-provisioner:latest
+      env:
+        - name: PROVISIONER_NAME
+          valueFrom:
+            configMapKeyRef:
+              name: efs-provisioner
+              key: provisioner.name
+        - name: FILE_SYSTEM_ID
+          valueFrom:
+            configMapKeyRef:
+              name: efs-provisioner
+              key: file.system.id
+        - name: AWS_REGION
+          valueFrom:
+            configMapKeyRef:
+              name: efs-provisioner
+              key: aws.region
+        - name: DNS_NAME
+          valueFrom:
+            configMapKeyRef:
+              name: efs-provisioner
+              key: dns.name
+              optional: true
+      volumeMounts:
+        - name: pv-volume
+          mountPath: /persistentvolumes
+  volumes:
+    - name: pv-volume
+      nfs:
+        server: <file-system-id>.efs.<region>.amazonaws.com <1>
+        path: / <2>
+----
+<1> Contains the DNS name of the EFS volume. This must be updated
+for the Pod to discover the EFS volume.
+<2> The mount path of the EFS volume. Each persistent volume is created
+as a separate subdirectory on the EFS volume. If this EFS volume is used
+for other projects outside of {product-title}, then it is recommended to
+create a unique subdirectory {product-title} manually on EFS for the
+cluster. Specifying a directory that does not exist will result in an error.
+
+. Once the file has been configured, it may be created in your cluster
```

Comment:
s/Once/After

! once adv
Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”

Do you need to do this to proceed? If so, s/it may be created/create it